cluster.go

```diff
@@ -898,14 +957,30 @@ func askingMulti(cc conn, ctx context.Context, multi []Completed) *redisresults
 	return results
 }
 
-func askingMultiCache(cc conn, ctx context.Context, multi []CacheableTTL) *redisresults {
+func (c *clusterClient) askingMultiCache(cc conn, ctx context.Context, multi []CacheableTTL) *redisresults {
 	commands := make([]Completed, 0, len(multi)*6)
 	for _, cmd := range multi {
 		ck, _ := cmds.CacheKey(cmd.Cmd)
 		commands = append(commands, cc.OptInCmd(), cmds.AskingCmd, cmds.MultiCmd, cmds.NewCompleted([]string{"PTTL", ck}), Completed(cmd.Cmd), cmds.ExecCmd)
 	}
 	results := resultsp.Get(0, len(multi))
 	resps := cc.DoMulti(ctx, commands...)
+	if c.hasLftm {
+		var ml []Completed
+	recover:
+		ml = ml[:0]
+		for i, resp := range resps.s {
```

Comment:
Here, we can advance the `i` by 6 in every iteration and only check the Error of the ExecCmd.

Reply:
Fix https://github.com/redis/rueidis/pull/833/commits/6362824f8c826e8c96313ea41fb92eb51eddf7c6